README.md

```diff
@@ -25,10 +25,12 @@ be adapted before any real usage.
 
 To use our Cassandra template:
 
+    kubectl create -f https://raw.githubusercontent.com/jaegertracing/jaeger-kubernetes/master/production/configmap.yml
```

Comment:
I am not super happy about this, and I think people are not using it this way, they probably clone it and run it from localhost. 

kubectl supports something like this `kubectl create -f dir` and it loads all resources form that dir. I am in a favor of changing it to local version of the command.

Reply:
How does it work? If I have `configmap.yaml` and an `application.yaml` referencing the ConfigMap created on the `configmap.yaml`, will it work, or will it fail because the ConfigMap might not exist yet?

Reply:
I don't know how it is implemented, I have tried this and it worked. Docs also say that <FILE> can be a directory, I expect there is some logic to process files in a "valid" order.

Reply:
Let me know if you think it's better to change the current commands